src/shared/components/mutationMapper/MutationMapper.tsx

```diff
@@ -90,6 +90,8 @@ export default class MutationMapper<
         makeObservable(this);
     }
 
+    private driverVusVusSelectorComponent: DriverAnnotationProteinImpactTypeBadgeSelector | null = null;
```

Comment:
I guess there is a typo here:

```suggestion
    private driverVsVusSelectorComponent: DriverAnnotationProteinImpactTypeBadgeSelector | null = null;
```

But actually, I would prefer to handle driver vs vus logic in the filter applier and derive everything from the filter itself. We wouldn't need a reference like this in that case.